test_runner/regress/test_tenant_delete.py

```diff
@@ -556,6 +556,15 @@ def hit_run_failpoint():
     assert ps_http.get_metric_value("pageserver_tenant_manager_slots") == 0
 
 
+@pytest.mark.parametrize("_nth", range(30))
```

Comment:
This is temporary and only exists to prove that the issue is gone. Will remove it after a successful CI run.

Reply:
[link](https://neon-github-public-dev.s3.amazonaws.com/reports/pr-6451/7629755887/index.html) to successful run